packages/kbn-config-schema/src/typeguards/is_config_schema.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { Type } from '../types';
+
+export function isConfigSchema(obj: any): obj is Type<any> {
+  return obj ? obj.__isKbnConfigSchemaType === true : false;
```

Comment:
I started with the `?.` syntax, but it looks like the `packages` are not transpiled the same way `src` is, and I was getting CI failures with

```
return obj?.__isKbnConfigSchemaType === true;
               ^

SyntaxError: Unexpected token .
```

in karma tests...